upgrade/steps.go

```diff
@@ -535,6 +537,8 @@ var getWorkingBranch = stepv2.Func41E("Working Branch Name", func(ctx context.Co
 		return ret("upgrade-pulumi-version-to-%s", c.TargetPulumiVersion)
 	case c.UpgradeJavaVersion:
 		return ret("upgrade-java-version-to-%s", c.JavaVersion)
+	case c.UpgradeTFSDKVersion:
+		return ret("upgrade-tf-sdk-version-to-%s", c.TargetTFSDKRef)
```

Comment:
As I understand it, we will only hit this case when performing a bridge upgrade where the bridge is already up to date but a new version of the TF SDK has been released. Is that correct?

I'm not sure we actually want to upgrade the TF SDK in that case. It seems like we only need to trigger a TF SDK upgrade when we are upgrading the bridge.